packages/nouns-webapp/src/config.ts

```diff
@@ -70,15 +70,13 @@ const app: Record<SupportedChains, AppConfig> = {
   [sepolia.id]: {
     jsonRpcUri: createNetworkHttpUrl('sepolia'),
     wsRpcUri: createNetworkWsUrl('sepolia'),
-    subgraphApiUri:
-      'https://api.goldsky.com/api/public/project_cldf2o9pqagp43svvbk5u3kmo/subgraphs/nouns-sepolia-the-burn/0.1.0/gn',
+    subgraphApiUri: import.meta.env.VITE_SEPOLIA_SUBGRAPH ?? '',
     enableHistory: import.meta.env.VITE_ENABLE_HISTORY === 'true',
   },
   [mainnet.id]: {
     jsonRpcUri: createNetworkHttpUrl('mainnet'),
     wsRpcUri: createNetworkWsUrl('mainnet'),
-    subgraphApiUri:
-      'https://api.goldsky.com/api/public/project_cldf2o9pqagp43svvbk5u3kmo/subgraphs/nouns/prod/gn',
+    subgraphApiUri: import.meta.env.VITE_MAINNET_SUBGRAPH ?? '',
```

Comment:
can you add those to .env.example(.local)

Reply:
Added `VITE_MAINNET_SUBGRAPH` and `VITE_SEPOLY_SUBGRAPH` to `packages/nouns-webapp/.env.example.local`
Also updated the deployment scripts in `packages/nouns-subgraph/package.json` so you (or anyone) can spin up Mainnet or Sepolia subgraphs on The Graph for testing and dev
Just update your {user_id} in `VITE_MAINNET_SUBGRAPH` and `VITE_SEPOLIA_SUBGRAPH` after you deploy your subgraphs 🤘